### ISSUE TITLE
Display fp & path for HD watch-only

### ISSUE THREAD

**Issue by Overtorment** [BlueWallet/BlueWallet#3371]
AFAIR we already display fp (but only in advanced mode), but not path.

**Comment by demerzel3** [comment]
how can I create a watch only HD wallet with a specific derivation path? I can import an xpub but I can see no way to specify the path. what am I missing?

**Comment by Overtorment** [comment]
you can import watch-only by pairing with keystone or foundation HW wallets. the feed to bluewallet structure like 

```
{"ExtPubKey":"zpub6riZchHnrWzhhZ3Z4dhCJmesGyafMmZBRC9txhnidR313XJbcv4KiDubderKHhL7rMsqacYd82FQ38e4whgs8Dg7CpsxX3dSGWayXsEerF4","MasterFingerprint":"7D2F0272","AccountKeyPath":"84'\/0'\/0'","CoboVaultFirmwareVersion":"2.6.1(BTC-Only)"}
```

which has path.

in other cases when we do not have path on hand (when we imported plain xpub string) is somewhat safe to assume its default BIP84 path for native segwit, BIP49 path for wrapped segwit  and BIP44 path for legacy

**Comment by demerzel3** [comment]
@Overtorment another quick question for you: I see an `additionalProperties` argument in a couple of places in the code that is supposed to hold the value for the master fingerprint for watch only wallets. Here's [one place](https://github.com/demerzel3/BlueWallet/blob/038890a1a9912b6d5d3ed7e4e12f3edd4e499bd8/class/wallet-import.js#L95), there are a couple more.

From what I can see though, the master fingerprint for watch only addresses is set in abstract wallet [here](https://github.com/demerzel3/BlueWallet/blob/038890a1a9912b6d5d3ed7e4e12f3edd4e499bd8/class/wallets/abstract-wallet.js#L166-L221) inside of `setSecret`.

In short, I can't see `additionalProperties` being used at all, am I missing something or is it safe to remove?

**Comment by Overtorment** [comment]
i dont think that `additionalProperties` is used anymore, can cautiously remove it
